test/controllers/mass_rollouts_controller_test.rb

```diff
@@ -59,193 +59,113 @@ def create_deploy(stage, reference, status)
       end
     end
 
-    describe "#deploy" do
-      def deploy(deploy_group)
-        post :deploy, params: {deploy_group_id: deploy_group, status: status}
+    describe "#review_deploy" do
+      def review_deploy(options = {})
+        get :review_deploy, params: {deploy_group_id: deploy_group}.merge(options)
       end
 
-      let(:status) { nil }
-
-      it "deploys all" do
-        assert_difference('Deploy.count', 1) do
-          post :deploy, params: {deploy_group_id: deploy_groups(:pod100)}
-          assert_response :redirect
-        end
+      it "can drive list from console by building a url with needed ids" do
```

Comment:
i have no idea what that test description means.. how about just deploys with list of stage ids.

Reply:
I'm trying to explain the usecase, which is generating list of stage-ids from a console sessions (for secrets migration)